CHANGELOG.rst

```diff
@@ -16,6 +16,15 @@ Unreleased
 ~~~~~~~~~~
 
 
+[3.7.2] - 2020-08-10
+~~~~~~~~~~~~~~~~~~~~
+* plugins.__init__.py was missing some objects used by users, this release adds:
+    - PluginSignals, PluginContexts, PluginError
+
+* also changing function names to be more descriptive
+
+.. note:: these changes are backwards incompatible. This is being put out as a bug fix because plugins code release(3.7.0) is not yet in use.
```

Comment:
Updated changelog entry:
```
Updated
_______

* Added missing classes to plugins public api. See ``plugins.__init__.py`` for latest api.
* Updated plugin method names to be more descriptive. See ``plugins.__init__.py`` for latest.

.. note:: Although these changes are backwards incompatible, they are being added as a bug fix because plugins code release (3.7.0) is not yet in use.
```

Reply:
Note: 3.7.0 and 3.7.1 changelog could be improved as well, by using `Added` header for the first and `Updated` for the second. Also, the sub-bullets for each of those aren't correct for rST.  Simplest is just making it one long line with both sentences.  Or, you can do:
```
* Bullet
  
  * Sub-bullet

* Another bullet
```